src/viam/sdk/common/exception.cpp

```diff
@@ -0,0 +1,61 @@
+#include <viam/sdk/common/exception.hpp>
+
+namespace viam {
+namespace sdk {
+
+Exception::Exception(const std::string& what)
+    : std::runtime_error("ViamException: " + what), condition_(ErrorCondition::k_general){};
```

Comment:
Maybe better to use constructor delegation here.

Reply:
Yeah good point; done.